eZ/Publish/Core/QueryType/QueryParameterContentViewQueryTypeMapper.php

```diff
@@ -61,13 +61,22 @@ private function extractParameters(ContentView $contentView)
 
     /**
      * @param ContentView $contentView
-     * @param string $queryParameterValue
+     * @param string|array $queryParameterValue
      *
      * @return mixed
      */
     private function evaluateExpression(ContentView $contentView, $queryParameterValue)
     {
-        if (substr($queryParameterValue, 0, 2) === '@=') {
+        if (is_array($queryParameterValue)) {
```

Comment:
This is already a good improvement.

Maybe I'd like it even better if `evaluateExpression` was only in charge of, well, _evaluating expressions_ (it does more than that with this code). Could you try to suggest something in that direction (the methods are private, we can fiddle with them).

About the fix version for this, we agree that v1.7.0 isn't even considered, do we ?